src/runtime_src/core/common/drv/xrt_cu.c

```diff
@@ -514,9 +514,10 @@ ssize_t show_cu_stat(struct xrt_cu *xcu, char *buf)
 	sz += sprintf(buf+sz, "Running queue:    %d\n", xcu->num_rq);
 	sz += sprintf(buf+sz, "Submitted queue:  %d\n", xcu->num_sq);
 	sz += sprintf(buf+sz, "Completed queue:  %d\n", xcu->num_cq);
-	sz += sprintf(buf+sz, "Bad state:	%d\n", xcu->bad_state);
-	sz += sprintf(buf+sz, "Current credit:  %d\n",
+	sz += sprintf(buf+sz, "Bad state:        %d\n", xcu->bad_state);
```

Comment:
Again, potential issue with buffer overruns.  See earlier comment on how to address this.  In fact, you might wish to create a helper method that will do these checks for you.